javascript/validatepassword.js

```diff
@@ -0,0 +1,26 @@
+/*
+    validatepassword.js
+
+    MediaWiki Action API Code Samples
+    Demo of `validate password` module: Validate a password against the wiki's password policies.
```

Comment:
Let's make this " Demo of `Validatepassword` module: Validate a password against the wiki's password policies." Make this change if the other files as well